charts/rucio-daemons/values.yaml

```diff
@@ -398,7 +398,24 @@ ftsRenewal:
     pullPolicy: Always
   vo: "cms"
   voms: "cms:/cms/Role=production"
+  gridPassphrase:
+   required: false
+   existingSecret:
+     name: 'my-explicit-secret-name'
```

Comment:
Everything looks good to me. But `my-explicit-secret-name` as a default value seems wrong to me :) maybe call it `grid-passphrase`? Same for the ones bellow.